<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Notificaciones</class>
 <widget class="QWidget" name="Notificaciones">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Notificaciones</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
    background-color: #f8f9fa;
    font-family: 'Segoe UI', Arial, sans-serif;
}

QLabel {
    color: #333;
    font-weight: 500;
}

QPushButton {
    background-color: #007bff;
    color: white;
    border: none;
    border-radius: 6px;
    padding: 8px 16px;
    font-weight: 500;
    min-height: 20px;
}

QPushButton:hover {
    background-color: #0056b3;
}

QPushButton:pressed {
    background-color: #004085;
}

QPushButton#btnMarcarLeidas {
    background-color: #28a745;
}

QPushButton#btnMarcarLeidas:hover {
    background-color: #1e7e34;
}

QPushButton#btnEliminarTodas {
    background-color: #dc3545;
}

QPushButton#btnEliminarTodas:hover {
    background-color: #c82333;
}

QScrollArea {
    border: 1px solid #dee2e6;
    border-radius: 8px;
    background-color: white;
}

QScrollArea QWidget {
    background-color: white;
}

QFrame.notificacion-item {
    background-color: white;
    border: 1px solid #e9ecef;
    border-radius: 8px;
    margin: 4px;
    padding: 12px;
}

QFrame.notificacion-item:hover {
    background-color: #f8f9fa;
    border-color: #007bff;
}

QFrame.notificacion-nueva {
    background-color: #e3f2fd;
    border-color: #2196f3;
}

QLabel.titulo-notificacion {
    font-size: 14px;
    font-weight: 600;
    color: #2c3e50;
    margin-bottom: 4px;
}

QLabel.mensaje-notificacion {
    font-size: 13px;
    color: #555;
    line-height: 1.4;
}

QLabel.fecha-notificacion {
    font-size: 11px;
    color: #6c757d;
    font-style: italic;
}

QLabel.tipo-notificacion {
    font-size: 10px;
    font-weight: 600;
    padding: 2px 8px;
    border-radius: 12px;
    color: white;
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>16</number>
   </property>
   <property name="leftMargin">
    <number>20</number>
   </property>
   <property name="topMargin">
    <number>20</number>
   </property>
   <property name="rightMargin">
    <number>20</number>
   </property>
   <property name="bottomMargin">
    <number>20</number>
   </property>
   <item>
    <widget class="QLabel" name="labelTitulo">
     <property name="text">
      <string>Centro de Notificaciones</string>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
    font-size: 24px;
    font-weight: 700;
    color: #2c3e50;
    margin-bottom: 8px;
}</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="layoutControles">
     <property name="spacing">
      <number>12</number>
     </property>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnActualizar">
       <property name="text">
        <string>Actualizar</string>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
    background-color: #6c757d;
}

QPushButton:hover {
    background-color: #545b62;
}</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnMarcarLeidas">
       <property name="text">
        <string>Marcar como leÃ­das</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnEliminarTodas">
       <property name="text">
        <string>Eliminar todas</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QScrollArea" name="scrollAreaNotificaciones">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">QScrollArea {
    border: 1px solid #dee2e6;
    border-radius: 8px;
    background-color: white;
}

QScrollBar:vertical {
    border: none;
    background-color: #f8f9fa;
    width: 12px;
    border-radius: 6px;
}

QScrollBar::handle:vertical {
    background-color: #ced4da;
    border-radius: 6px;
    min-height: 20px;
}

QScrollBar::handle:vertical:hover {
    background-color: #adb5bd;
}

QScrollBar::add-line:vertical, QScrollBar::sub-line:vertical {
    border: none;
    background: none;
}

QScrollBar::add-page:vertical, QScrollBar::sub-page:vertical {
    background: none;
}</string>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>758</width>
        <height>69</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="layoutNotificaciones">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="labelSinNotificaciones">
         <property name="text">
          <string>ðŸ“­ No hay notificaciones disponibles</string>
         </property>
         <property name="styleSheet">
          <string notr="true">QLabel {
    font-size: 16px;
    color: #6c757d;
    padding: 40px;
    text-align: center;
}</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
